src/target/portfolio-1/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```

Comment:
This file doesn't seem to be needed in the maptivist project? Was it added accidentaly?

Reply:
Ah - I think we need to add a .gitignore file that excludes the maven output. See this stackoverflow article:

https://stackoverflow.com/questions/51792830/what-files-in-a-maven-project-should-be-committed-to-git

Reply:
Oh thank you, I wasn't sure about this.

Reply:
Oh I see, it was taken off when the portfolio was copied over